packages/swapper/src/swappers/cow/utils/constants.ts

```diff
@@ -15,3 +15,7 @@ export const ORDER_KIND_BUY = 'buy'
 export const SIGNING_SCHEME = 'ethsign'
 export const ERC20_TOKEN_BALANCE = 'erc20'
 export const ORDER_STATUS_FULFILLED = 'fulfilled'
+
+// Address used by CowSwap to buy ETH
+// See https://github.com/gnosis/gp-v2-contracts/commit/821b5a8da213297b0f7f1d8b17c893c5627020af#diff-12bbbe13cd5cf42d639e34a39d8795021ba40d3ee1e1a8282df652eb161a11d6R13
+export const COW_SWAP_ETH_BUY_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
```

Comment:
I would prefer to see the mark address terminology vs. buy address to honor the one in
https://github.com/gnosis/gp-v2-contracts/pull/316

Technically, you're not buying any ERC20 at this address because there is no contract deployed at this address and there never will, it's just used as a mark.

Reply:
this is a kinda cool yet terrifying implementation at the same time

Reply:
@gomesalexandre  done !